src/index.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+// import _, { isLength } from 'lodash';
+
+import './style.css';
+import Book from '../module/bookClass.js';
+import { remove, bookListDeleted } from '../module/remove.js';
+import { clearAllTask, notCompleted } from '../module/checkbox.js';
+
+let description;
+let completed;
+let index;
+let bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+let bookgenerator = '';
+const taskList = document.getElementById('taskList');
+const input = document.getElementById('myInput');
+
+function getTask() {
+  description = input.value;
+  completed = false;
+  index = bookList.length + 1;
+}
```

Comment:
I suggest that instead of declaring the values of the object in a separate function you can declare the values in the class that creates the object, in your case the class Book